bindings/python/test.py

```diff
@@ -266,6 +268,34 @@ def test_apply_settings(self):
 		self.assertEqual(s.get_settings()['num_want'], 66)
 		self.assertEqual(s.get_settings()['user_agent'], 'test123')
 
+class test_example_client(unittest.TestCase):
+
+	def test_execute_client(self):
+		with open(os.devnull, "w") as DEVNULL:
+			process = subprocess.Popen([sys.executable,"client.py","url_seed_multi.torrent"], stdout=DEVNULL, stderr=subprocess.STDOUT)
+		# python2 has no Popen.wait() timeout
+		time.sleep(10)
+		returncode = process.poll()
+		if returncode == None:
+			process.kill()
+		else:
+			self.assertEqual(returncode, 0)
+
+	def test_execute_simple_client(self):
+		with open(os.devnull, "w") as DEVNULL:
+			process = subprocess.Popen([sys.executable,"simple_client.py","url_seed_multi.torrent"], stdout=DEVNULL, stderr=subprocess.STDOUT)
+		# python2 has no Popen.wait() timeout
+		time.sleep(10)
+		returncode = process.poll()
```

Comment:
I'm pretty confident you could call: ``process.wait()`` here. in the case of ``client.py`` though, you still want to wait some time before sending a `q`. But I just realized you're running ``simple_client.py``, not ``client.py``. It would probably make sense to run client.py too